reference/conanfile/tools/files/basic.rst

```diff
@@ -356,3 +356,46 @@ Example:
     
         def source(self):
             data = self.conan_data["sources"]["mydata"]
+
+
+conan.tools.files.collect_libs()
+--------------------------------
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases/tag/1.46.0>`_
+
+.. code-block:: python
+
+    def collect_libs(conanfile, folder=None)
+
+Returns a sorted list of library names from the libraries (files with extensions *.so*, *.lib*, *.a* and *.dylib*) located inside the
+``conanfile.cpp_info.libdirs`` (by default) or the **folder** directory relative to the package folder. Useful to collect not
```

Comment:
I am not sure I understand the "folder directory relative to the package folder"

Does this mean `lib/` `bin/` dirs in the `conanfile.packacage_folder`?